pkg/storaged/crypto/keyslots.jsx

```diff
@@ -78,17 +79,18 @@ function clevis_unlock(block) {
                          { superuser: true });
 }
 
-export function unlock_with_type(client, block, passphrase, passphrase_type) {
+export async function unlock_with_type(client, block, passphrase, passphrase_type) {
     const crypto = client.blocks_crypto[block.path];
-    if (passphrase)
-        return crypto.Unlock(passphrase, {});
-    else if (passphrase_type == "stored")
-        return crypto.Unlock("", {});
-    else if (passphrase_type == "clevis")
-        return clevis_unlock(block);
-    else {
+    if (passphrase) {
+        await crypto.Unlock(passphrase, {});
+        remember_passphrase(block, passphrase);
+    } else if (passphrase_type == "stored") {
+        await crypto.Unlock("", {});
+    } else if (passphrase_type == "clevis") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19924-20240212-163533-e482442c-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)